draft-ietf-dprive-dnsoquic.md

```diff
@@ -971,19 +970,19 @@ split into three regions that are governed by different policies:
 
 * Permanent registrations for values between 0x00 and 0x3f (in hexadecimal;
 inclusive), which are assigned using Standards Action or IESG Approval as
-defined in Section 4.9 and 4.10 of {{!RFC8126}}
+defined in {{Section 4.9 of RFC8126}} and {{Section 4.10 of RFC8126}}
```

Comment:
```suggestion
defined in {{Sections 4.9 and 4.10 of RFC8126}}
```

Yes, it is smart enough to do that.

Reply:
Thanks for the pointer :-)

Reply:
Minor nit (and probably known issue) with the tooling I imagine:
This works beautifully in the html: 
https://www.ietf.org/archive/id/draft-ietf-dprive-dnsoquic-06.html#name-dns-over-quic-error-codes-r
but gets the first part wrong in the htmlized version:
https://datatracker.ietf.org/doc/html/draft-ietf-dprive-dnsoquic-06#section-10.4

Reply:
+1 to ditching the htmlized version - it seems to have a bunch or problems and the HTML is much nicer. Thanks for your work on this.